pkg/morserino/channels.go

```diff
@@ -22,30 +22,25 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 */
 
-import (
-	"fmt"
-	"strings"
-)
+// Structure containing all the channels used by the application
+type MorserinoChannels struct {
```

Comment:
Design nit: What is this structure role? What is its responsibility in regard of your application logic? 
Usually we try  to have one structure, one behavior. Here it seems that this structure is a technical factorization,  not a semantic one which isn't the best.


Alsoooo,  is repeating `Morserino` really necesasry here? We're already under the `morserino` package :smile: